### DISCUSSION TITLE
How to code

### DISCUSSION THREAD

**Discussion by irhrhd** [Prodigy-Hacking/ProdigyMathGameHacking#571] [(A) Anything]
I would like to learn java and typescript so i can help code this stuff i think coding is pretty cool so any mod could get back to me that would be great

**Comment by ArcerionDev** [comment]
https://freecodecamp.com
https://javascript.info

You can learn the fundamentals from here
take a look at the objects in the chrome console to get an understanding of the prodigy object hierarchy

**Reply by irhrhd** [reply]
Its working now and its fun to do thank you ill be able to help you guys *in* edited from im to in a few weeks with coding

**Comment by irhrhd** [comment]
Free code camp isnt working

**Comment by ghost** [comment]
Erm... I've taught a class of 3 people before, but they got bored once I tried to move from Scratch Blocks to Python. I could try to help you?

**Comment by irhrhd** [comment]
Oh yeah help please i think you could im learning js right now

**Reply by ghost** [reply]
I suggest that you learn Python first because it's one of the easiest languages. Then you should learn TypeScript and Javascript, then learn even more languages if you're interested.

**Comment by irhrhd** [comment]
Okay sure but pleaseee not scratch

**Reply by ghost** [reply]
Okie.

**Reply by ghost** [reply]
What type of device do you use?

If the bottom of the screen fills the entire screen width and shows 4 squares with "Type here to search" next to it, it's a Windows computer.

If you're using a Mac OS X device, it will look like this:
![Screenshot-2021-03-02-164311](https://user-images.githubusercontent.com/61570792/109734921-adf7fd00-7b76-11eb-8260-808ca9726a61.png)

If you're using a Chromebook, it will look like this:
![Screenshot 2021-03-02 164419](https://user-images.githubusercontent.com/61570792/109734852-90c32e80-7b76-11eb-8a54-dc4054e896e3.png)

**Comment by irhrhd** [comment]
Im on mobile ipad

**Reply by ghost** [reply]
it's honestly kinda hard to code on an ipad unless you have a physical keyboard

do you have a bluetooth keyboard that you can use?

**Comment by irhrhd** [comment]
Ya i use a keyboard

**Reply by ghost** [reply]
o ok thas good

ok gimme a moment

**Reply by ghost** [reply]
alright here ya go https://apps.apple.com/us/app/pythonista-3/id1085978097

**Reply by irhrhd** [reply]
Hey i have a code editor app now its a web app that i added to home screen

**Reply by ghost** [reply]
oo ok
